nogo: ignore generated source files (#3216)
bazelbuild/rules_go: tests/core/nogo/coverage/README.rst

gen_code_test
-------------

> Checks how `nogo`_ shoudl not interact with source code that was generated as part of

Reviewer: ```suggestion
Checks how `nogo`_ should not run on source code that was generated as part of
```
